openupgrade_scripts/scripts/project/14.0.1.1/post-migration.py

```diff
@@ -28,7 +28,7 @@ def _fill_res_users_m2m_tables(env):
         """
         INSERT INTO project_allowed_internal_users_rel
         (project_project_id, res_users_id)
-        SELECT pp.id, ru.id
+        SELECT DISTINCT pp.id, ru.id
```

Comment:
Here there's no commercial partner, and that's why I was confused, as I didn't see the second one. Is this still needed?

Reply:
no, just removed it